include/llzk/Analysis/SourceRef.h

```diff
@@ -198,18 +198,18 @@ class SourceRef {
   }
 
   llvm::DynamicAPInt getConstantFeltValue() const {
-    ensure(isConstantFelt(), __FUNCTION__ + mlir::Twine(" requires a constant felt!"));
+    ensure(isConstantFelt(), mlir::Twine(__FUNCTION__).concat(" requires a constant felt!"));
```

Comment:
There's a constructor in `Twine` that apparently is optimized for this particular case:

```c++
mlir::Twine(mlir::StringRef(__FUNCTION__), "....")
```

Constructs the `Twine` in such a way that basically avoids calling `concat`.

Docs: https://llvm.org/doxygen/classllvm_1_1Twine.html#aba6d9549c0ddfa25919915044a81eaf3